packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -163,7 +157,10 @@ export default class ImageEdit implements EditorPlugin {
      */
     initialize(editor: IEditor) {
         this.editor = editor;
-        this.disposer = editor.addDomEventHandler('blur', this.onBlur);
+        this.disposer = editor.addDomEventHandler({
+            blur: () => this.onBlur,
+            drop: e => e.preventDefault(),
```

Comment:
I think we should disable onDrag instead, and only disable when image is in editing.